shimming-toolbox/shimmingtoolbox/cli/mask.py

```diff
@@ -302,6 +302,49 @@ def sct(fname_input, fname_output, contrast, centerline, file_centerline, brain,
     click.echo(f"The path for the output mask is: {os.path.abspath(fname_output)}")
     return fname_output
 
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+                  help="Wrapper for BET, please see https://fsl.fmrib.ox.ac.uk/fsl/docs/#/structural/bet. "
+                    "Create a brain mask in the coordinates of the input file. The mask is stored by default "
+                       "under the name 'mask.nii.gz' in the output folder.")
+@click.option('-i', '--input', 'fname_input', type=click.Path(), required=True,
+              help="Input path of the nifti file to mask. This nifti file must be 3D. Supported "
+                   "extensions are .nii or .nii.gz.")
+@click.option('-o', '--output', 'fname_output', type=click.Path(), default=os.path.join(os.curdir, 'mask.nii.gz'),
+              show_default=True, help="Name of output mask. Supported extensions are .nii or .nii.gz.")
+@click.option('-f', '--f_param', required=False, type=float, default=0.5,
+              help="fractional intensity threshold (0->1); default=0.5; smaller values give larger brain outline estimates")
+@click.option('-g', '--g_param', type=float, required=False, default=0,
+              help="vertical gradient in fractional intensity threshold (-1->1); positive values give larger brain outline at bottom, smaller at top")
```

Comment:
I think you can use a click.FloatRange to validate the input

Reply:
bet will tell if the input is too much, do we really need to verify again?

Reply:
Fair